tracee-rules/signatures/rego/anti_debugging_ptraceme.rego

```diff
@@ -1,8 +1,8 @@
 package main
 
 __rego_metadoc__ := {
-    "name": "detect self debugging using PTRACE_ME",
-    "description": "Process uses anti-debugging technique to block debugger",
+    "name": "PTRACE_ME",
```

Comment:
I'm not sure about the name change.. this is supposed to be eventually displayed in user facing UIs (DTA/AVD/etc). If the intent is to make it accessible for selecting on the command like, #515 should fix that

Reply:
Ah alright, will revert

Reply:
Other sig as well?